File: csharp/darts/Darts.cs
using System;

public static class Darts
{
    public static int Score(double x, double y)
    => (Math.Sqrt(Math.Pow(x, 2) + Math.Pow(y, 2))) switch
    {
        <= 1 => 10,
        <= 5 => 5,
        <= 10 => 1,
        _ => 0,
    };
}

File: java/darts/src/main/java/Darts.java
class Darts {
    final int innerCircle = 1;
    final int middleCircle = 5;
    final int outerCircle = 10;

    int score(double xOfDart, double yOfDart) {
        double radius = Math.sqrt(xOfDart * xOfDart + yOfDart * yOfDart);

        if (radius <= innerCircle) {
            return 10;
        }
        if (radius <= middleCircle) {
            return 5;
        }
        if (radius <= outerCircle) {
            return 1;
        }
        return 0;
    }
}
